### ISSUE TITLE
README instructions are wrong?

### ISSUE THREAD

**Issue by soggie** [algolia/search-insights.js#222]
Hey guys, according to your README, it says:

```
const aa = require('search-insights')

aa('init', { ... })
```

However, aa is an object. Am I going crazy and missed something? I've scoured the documentation multiple times.

**Comment by eunjae-lee** [comment]
Hello @soggie, unfortunately we've discovered this issue yesterday.
For now, if you load the library with `require`, you should prepend `.default`.

```js
const aa = require('search-insights').default;

aa('init', { ... })
```

Sorry for inconvenience and I will make sure to fix this issue or at least update the documentation shortly.

**Comment by eunjae-lee** [comment]
Hello @soggie , it's been fixed in `1.6.2`.
